packages/okta-react-native/README.md

```diff
@@ -191,14 +191,38 @@ await createConfig({
 });
 ``` 
 
+### `getAuthClient`
+
+This method will return an instance of [`@okta/okta-auth-js`](https://github.com/okta/okta-auth-js) client to communicate with Okta Authentication API. For more information, please checkout [Okta AuthJs Node JS and React Native Usage](https://github.com/okta/okta-auth-js#node-js-and-react-native-usage) section.
+
 ### `signIn`
 
-This async method will automatically redirect users to your Okta organziation for authentication. It will emit an event once a user successfully signs in. Make sure your event listeners are mounted and unmounted. Note: on iOS there isn't a `onCancelled` event. If the sign in process is cancelled, `onError` will be triggered. 
+This method will handle both `browser-sign-in` and `custom-sign-in` scenarios based on provided options.
+
+This async method will automatically redirect users to your Okta organziation for authentication. It will emit an event once a user successfully signs in. Make sure your event listeners are mounted and unmounted. Note: on iOS there isn't a `onCancelled` event. If the sign in process is cancelled, `onError` will be triggered.
+
+#### `browser-sign-in`
+`browser-sign-in` leverages device's native browser to automatically redirect users to your Okta organziation for authentication. By providing no argument, this method will trigger the `browser-sign-in` flow. It will emit an event once a user successfully signs in. Make sure your event listeners are mounted and unmounted. **Note**: on iOS there isn't a `onCancelled` event. If the sign in process is cancelled, `onError` will be triggered. 
```

Comment:
Is this Note about iOS a bug or a feature?

Reply:
Sort of a bug to declare the inconsistency between ios and android, here I just copied it over.

Reply:
Is this something we can easily fix? Would be nice to have consistent behavior, as long as we are releasing a new version may as well squeeze some fixes in.

Reply:
This is a inconsistency between android and iOS. I think in the iOS Bridge you can check if the error is a cancel and then call onCancelled. @IldarAbdullin-okta

Reply:
@IldarAbdullin-okta is the `onCancelled` event currently supported in the ios-oidc SDK?

Reply:
@shuowu , there is no way programmatically cancel auth flow. If user cancels manually then even will be delivered as error via `onError` callback

Reply:
Agree with @FeiChen-okta , that probably we can analyze error type in bridge and redirect to `onCancelled` callback if user manually cancelled

Reply:
Dug a bit deeper into the bridge and ios SDK's code, right now only throw a general error for cancel action, which is not enough to trigger the onCancelled event listener. Probably will need a bit investigation on ios SDK side.

I would suggest tracking the issue in a separate ticket if it needs to be addressed.

Reply:
Yes, separate ticket is fine, as most likely additional work will be required on iOS OIDC SDK side